lxd/db/cluster/stmt.go

```diff
@@ -41,6 +43,10 @@ var stmts = map[int]string{} // Statement code to statement SQL text.
 // PreparedStmts is a placeholder for transitioning to package-scoped transaction functions.
 var PreparedStmts = map[int]*sql.Stmt{}
 
+// DqliteMaxParams is the maximum amount of SQL query parameters that are supported by dqlite.
+// See: https://github.com/canonical/dqlite/issues/377
+const DqliteMaxParams = driver.MaxTupleParams
```

Comment:
I don't think we need this at all now looking at how its used. We should let `go-dqlite` fail the query if we exceed the max supported argument length.